app/views/recipes/_form.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= bootstrap_form_with(model: recipe) do |form| %>
+  <% if recipe.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(recipe.errors.count, "error") %> prohibited this recipe from being saved:</h2>
+
+      <ul>
+        <% recipe.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <%= form.text_field :name %>
+  <%= form.label :preparation_time, style: "display: block" %>
+  <%= form.number_field :preparation_time %>
+  <%= form.label :cooking_time, style: "display: block" %>
+  <%= form.number_field :cooking_time %>
+  <%= form.text_area :description %>
+  <%= form.select :public, options_for_select([true, false]), class:"block appearance-none w-full bg-gray-200 border border-gray-200 text-gray-700 py-3 px-4 pr-8 rounded leading-tight focus:outline-none focus:bg-white focus:border-gray-500" %>
+  <%= form.submit %>
```

Comment:
- [ ] I see that you selected `true` by default when I am selecting `false`  I am getting this error I am suggesting making it a choice between `true` and `false`
<img width="1436" alt="Screen Shot 2023-02-11 at 15 14 56" src="https://user-images.githubusercontent.com/44745855/218260223-7e2526fc-2d74-43b3-b9e8-a90137a81538.png">